samples/dashboards/demo/vertical/demo.html

```diff
@@ -0,0 +1,117 @@
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/dashboards/datagrid.js"></script>
+<script src="https://code.highcharts.com/dashboards/dashboards.js"></script>
+<script src="https://code.highcharts.com/dashboards/modules/layout.js"></script>
+<script src="https://code.highcharts.com/modules/accessibility.js"></script>
+
+<div id="container">
+    <h1 id="title">Climate Change Over the Last 10 Decades: A Deep Dive into the Data</h1>
+    <p>
+        Understanding how the climate has changed over time can help us prepare for the future. By looking at the data
+        from the last 100 years, we can see patterns, learn what’s driving these changes, and start thinking about how
+        we can address them. Let’s break down the numbers, decade by decade, to better understand the impact climate
+        change has had on our planet.
+    </p>
+    <div id="data-grid"></div>
+    <p>
+        Now, let's take a closer look at what each of these columns means and how they’ve developed over time.
+    </p>
+
+    <h2>1. Global Average Temperature Increase (°C)</h2>
+    <span>
+        The global average temperature is a key indicator of climate change. This column shows how much the Earth's
+        temperature has increased compared to pre-industrial levels.
+        <p>
+            <b>- 1920-1950:</b> For the first half of the century, temperature increases were relatively small.
+            Industrial
+            activity was growing, but it hadn’t yet reached the levels that would drastically impact the climate.
+        </p>
+        <p>
+            <b>- 1950-1980:</b> After World War II, industries boomed, and fossil fuel use increased
+            significantly. By the
+            1970s, the global temperature had increased by 0.3°C. This may seem like a small number, but for the
+            planet’s
+            delicate balance, even slight temperature changes can cause big disruptions.
+        </p>
+        <p>
+            <b>- 1980-2019:</b> From the 1980s onward, the pace of temperature increase accelerated dramatically.
+            By the
+            2010s, the global temperature had increased by 0.8°C. This steady increase is directly linked to human
+            activity,
+            particularly the burning of fossil fuels like coal, oil, and natural gas.
+        </p>
+        <div class="cell" id="temp-chart"></div>
```

Comment:
To make the layout more "interesting" I would like this chart to be positioned to the right of the text. Should take up 50% of available width, with some margin to the left.

Add 20px padding to the div, and use same bg color as chart. Add border-radius: 5px.

Reply:
Can't see that the additional styling is applied here? Padding, background-color and border-radius?